### PR TITLE
Setup feedback view #43

### PR DESCRIPTION
**Pull request by nemodjuric**
Adding view, setting up authorization. And authentication.  
Related to #6 
Write to a test to visit Feedback view page. 
screen shot 2016-11-29 at 6 00 13 pm

screen shot 2016-11-29 at 11 00 15 am

### CODE REVIEW

**Comment by nemodjuric** [comment]
screen shot 2016-11-29 at 10 59 09 am

**Comment by nemodjuric** [comment]
screen shot 2016-11-29 at 11 00 15 am

File: app/controllers/feedback_controller.rb [hunk 1]
```diff
@@ -1,4 +1,10 @@
 class FeedbackController < ApplicationController
+  before_action :verify_access, except: [:new, :create]
```
Comment (amprokop) [inline app/controllers/feedback_controller.rb]: also verify authentication

Comment (amprokop) [inline app/controllers/feedback_controller.rb]: only authenticated users should be able to see the feedback app

**Comment by amprokop** [comment]
LGTM!

**Pull request merged**